### PR TITLE
[ADD] Added new module for google adwards: website_sale_adwords_conversion

### PR DESCRIPTION
**Pull request by krunal267**
Task URL:

https://www.druidoo.io/web#id=1924&action=367&model=project.task&view_type=form&menu_id=237

### CODE REVIEW

File: website_sale_adwords_conversion/models/website.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    @api.multi
+    def write(self, vals):
+        template_view = self.env.ref(
+            'website_sale_adwords_conversion.website_layout')
+        if template_view:
+            if 'google_adwards_conversoin_tracking' in vals:
+                template_view.active = \
+                    vals.get('google_adwards_conversoin_tracking')
+        return super().write(vals)
```
Comment (ivantodorovich) [inline website_sale_adwords_conversion/models/website.py]: Hmmm. I think the view should always be active.
Website views are tricky. You can have multiple websites, and each website has a copy of the view.

We shouldn't toggle this feature via activating/deactivating a view. The script element inside the view should be toggled via a `t-if="website.google_adwords_conversion_key"` instead

File: website_sale_adwords_conversion/static/src/xml/payment_processing.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="payment" xml:space="preserve">
+
+    <t t-extend="payment.display_tx_list">
+        <t t-jquery="t[t-foreach*='tx_done'][t-as*='tx']" t-operation="before">
+            <t t-if="website.google_adwards_conversoin_tracking">
+                <t t-set="tx" t-if="tx_done.length > 0" t-value="tx_done[0]"/>
```
Comment (ivantodorovich) [inline website_sale_adwords_conversion/static/src/xml/payment_processing.xml]: I'm wondering what happens if there are multiple transactions.
Do you know what's that case for?

I don't remember odoo supporting multiple payments for the same order :thinking:

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
Thanks @krunal267 

a few comments.
Also, would you please add a readme?

File: website_sale_adwords_conversion/static/src/xml/payment_processing.xml (hunk 2 above)
Comment (krunal267) [reply website_sale_adwords_conversion/static/src/xml/payment_processing.xml]: Okay, let me see.

**Comment by krunal267** [comment]
Hi @ivantodorovich 

Suggested changes are pushed.

**Pull request merged**